lib/JSON/Validator/OpenAPI/Mojolicious.pm

```diff
@@ -19,6 +19,28 @@ has version => 2;
 
 sub E { JSON::Validator::Error->new(@_) }
 
+has generate_definitions_path => sub {
```

Comment:
I don't think this method has any pod. I'm good with just a reference to JSON::Validator, but please elaborate if you want to.

```
=head2 generate_definitions_path

See L<JSON::Validator/generate_definitions_path>.
```

Reply:
fixed in upcoming push